### PR TITLE
[#12375] Incorrect Developer Website link in Contact page 

### LINKED ISSUE #12375: Incorrect Developer Website link in Contact page
**Environment**: 
V8.26.0

**Steps to reproduce**

https://teammatesv4.appspot.com/web/front/contact

image

**Expected behaviour**

Should link to https://teammates.github.io/teammates/index.html.

**Actual behaviour**

Links to back to [TEAMMATES repository](https://github.com/TEAMMATES/teammates).

### PR DESCRIPTION
**Pull request by rahulpro2001**
Fixes #12375 

**Outline of Solution**

updating hyperlink

### CODE REVIEW

**Comment by rahulpro2001** [comment]
@weiquu kindly review and  merge my PR

**Comment by weiquu** [comment]
@domlimm is this consistent with your [suggestions](https://github.com/TEAMMATES/teammates/issues/12375#issuecomment-1508081571)?

File: src/web/app/pages-static/contact-page/contact-page.component.html [hunk 1]
```diff
@@ -12,7 +12,7 @@ <h1 class="color-orange">
   <b>Bug reports and feature requests:</b> Any bug reports or feature requests can be submitted to above email address.
 </p>
 <p>
-  <b>Interested in joining us?</b> Visit our <a href="https://github.com/TEAMMATES/teammates" target="_blank" rel="noopener noreferrer">Developer Website</a>.
+  <b>Interested in joining us?</b> Visit our <a href="https://teammates.github.io/teammates/index.html" target="_blank" rel="noopener noreferrer">Developer Website</a>.
```
Comment (domlimm) [inline src/web/app/pages-static/contact-page/contact-page.component.html:15]:
```suggestion
  <b>Interested in joining us?</b> Visit our <a href="https://github.com/TEAMMATES/teammates" target="_blank" rel="noopener noreferrer">GitHub</a> or <a href="https://teammates.github.io/teammates/index.html" target="_blank" rel="noopener noreferrer">Developer Documentation</a>.
```

**Review by domlimm — COMMENTED** [review]
Have added a comment on the change. Thank you for your contribution!

**Comment by weiquu** [comment]
Closing due to inactivity

**Pull request closed without merging**